docs/modules/ROOT/pages/how-tos/vsphere/pre-install-checklist.adoc

```diff
@@ -0,0 +1,175 @@
+
+|Webserver<<fn1,^1^>> |1 |1 |4 GB |20 GB |irrelevant
+
+|Bootstrap Node<<fn2,^2^>> |1 |4 |16 GB |120 GB |RHCOS<<fn4,^4^>>
+
+|Masters |3 |4 |16 GB |120 GB |RHCOS<<fn4,^4^>>
+
+|Infra Nodes |3 |8 |32 GB |120 GB |RHCOS<<fn4,^4^>>
+
+|Worker Nodes<<fn3,^3^>> |2+ |2+ |8+ GB |120 GB |RHCOS<<fn4,^4^>>
+|===
+
+[[fn1]]^1^: The bastion host can also be used as a web server. Red Hat does not define any resource requirements for bastion host and web server. The values above are based on our recommendations and can be adapted to your needs.
+
+[[fn2]]^2^: The bootstrap node is a temporary machine that can be deleted after the installation.
+
+[[fn3]]^3^: Number and dimensioning of the worker nodes is dependent on your workload.
+
+[[fn4]]^4^: RHCOS = Red Hat CoreOS
+
+== Network
+
+=== Address and DHCP
+
+The OpenShift VMs should get their IP address via DHCP. Create DHCP reservations with the specified MAC address of the VM.
+
+=== DNS records
+
+Several DNS records are required for OpenShift to work correctly.
+
+The https://docs.openshift.com/container-platform/4.4/installing/installing_vsphere/installing-vsphere-network-customizations.html#installation-dns-user-infra_installing-vsphere-network-customizations[OpenShift documentation] lists the required records in detail.
+
+To illustrate, the following variables are set:
+
+* `<base_domain>`: `appuio.ch`
+* `<cluster_name>`: `ocp4-vs`
+
+[CAUTION]
+====
+`<cluster_name>` must match the folder name in vSphere, in which all OpenShift VMs reside, for vSphere storage integration to work correctly.
+====
+
+With the defined variables, the following DNS records are required:
+
+|===
+|Record |Type |Target 
+
+|`api.ocp4-vs.appuio.ch` |A/CNAME |Console Load Balancer
+
+|`api-int.ocp4-vs.appuio.ch` |A/CNAME |Console Load Balancer
+
+|`*.apps.ocp4-vs.appuio.ch` |A/CNAME |Apps Load Balancer
+
+|`console.ocp4-vs.appuio.ch` |CNAME |Apps Load Balancer
+
+|`etcd-0.ocp4-vs.appuio.ch` |A |Master 1
+
+|`etcd-1.ocp4-vs.appuio.ch` |A |Master 2
+
+|`etcd-2.ocp4-vs.appuio.ch` |A |Master 3
+
+|`_etcd-server-ssl._tcp.ocp4-vs.appuio.ch` |SRV |See https://docs.openshift.com/container-platform/4.4/installing/installing_vsphere/installing-vsphere-network-customizations.html#installation-dns-user-infra_installing-vsphere-network-customizations[documentation] for details.
+|===
+
+[WARNING]
+====
+By default, Windows DNS servers create a SRV record for every A record.
+For the A records of `etcd`, make sure that the SRV records are not created (or delete them, if they were), so the hostnames can be enumerated correctly.
+====
+
+=== Certificates
+
+Certificates for the following hostnames need to be created:
+
+* `api.ocp4-vs.appuio.ch`
+* `console.ocp4-vs.appuio.ch`
+* `*.apps.ocp4-vs.appuio.ch` (wildcard certificate)
+
+[WARNING]
+====
+For the certificates to be valid in modern web browser, make sure the _Subject Alternative Name (SAN)_ contains the hostname of the certificate.
+
+[[additional-ca]]
```

Comment:
Might be worth to place the part about CA in its own admonition.

Reply:
I had it that way before, looked kinda odd.

Reply:
How so?

Reply:
I did not use the title before. Added title, looks good now!